--- ../../RVC/src/devel/org/sc29/wg11/mpegh/part2/inter/InterPrediction.cal	2013-02-08 16:59:00.418273535 +0100
+++ ../src/interPrediction/InterPredPipeline1.cal	2013-02-08 17:00:30.794275774 +0100
@@ -59,13 +59,13 @@
  * @author Damien de Saint Jorre (EPFL) <damien.desaintjorre@epfl.ch>
  */
 
-package devel.org.sc29.wg11.mpegh.part2.inter;
+package interPrediction;
 
 import std.util.Math.*;
 import devel.org.sc29.wg11.mpegh.part2.common.CommonConstantHEVC.*;
 import devel.org.sc29.wg11.mpegh.part2.inter.NaluType.*;
 
-actor InterPrediction(uint(size=2) DECODE_COMP, uint(size=2) PROCESS_MODE_HALF_PIX,
+actor InterPredPipeline1(uint(size=2) DECODE_COMP, uint(size=2) PROCESS_MODE_HALF_PIX,
 		uint(size=8) LOG2_NUM_SUB_BLK[2], uint(size=8) IDX_SUB_BLOCK[2])
 
 	uint(size=16) CUInfo,
@@ -79,7 +79,7 @@
 	uint(size=2)  SliceType,
 	int (size=16) WeightedPredSe // is size correct ?
 		==>
-	uint(size=8) PredSample
+	int(size=16) PredSample
 		:
 
 	uint(size=2) HALF_PIX_1ST_PART = 1;
