--- ../../RVC/src/devel/org/sc29/wg11/mpegh/part2/inter/InterPrediction.cal	2013-02-06 16:37:59.671693953 +0100
+++ ../src/interPrediction/InterPredPipeline1.cal	2013-02-06 17:03:54.163721474 +0100
@@ -59,13 +59,13 @@
  * @author Damien de Saint Jorre (EPFL) <damien.desaintjorre@epfl.ch>
  */
 
-package devel.org.sc29.wg11.mpegh.part2.inter;
+package interPrediction;
 
 import std.util.Math.*;
 import devel.org.sc29.wg11.mpegh.part2.common.CommonConstantHEVC.*;
 import devel.org.sc29.wg11.mpegh.part2.inter.NaluType.*;
 
-actor InterPrediction(bool DECODE_LUMA, bool DECODE_CHROMA, uint(size=2) PROCESS_MODE_HALF_PIX)
+actor InterPredPipeline1(bool DECODE_LUMA, bool DECODE_CHROMA, uint(size=2) PROCESS_MODE_HALF_PIX)
 	uint(size=16) CUInfo,
 	uint(size=2)  IsBiPredOrLx,
 	int (size=16) Mv,
@@ -77,7 +77,7 @@
 	uint(size=2)  SliceType,
 	int (size=16) WeightedPredSe // is size correct ?
 		==>
-	uint(size=8) PredSample
+	int(size=16) PredSample
 		:
 
 	uint(size=2) HALF_PIX_1ST_PART = 1;
